payments_service/base/tests/__init__.py

```diff
@@ -29,6 +30,17 @@ def tolist(self, *args, **kw):
 
 class TestCase(DjangoTestCase):
 
+    def setUp(self):
+        super(TestCase, self).setUp()
+
+        # Since all endpoints pretty much use Solitude, set up the mock
+        # for convenience.
+        p = mock.patch('payments_service.solitude.api')
```

Comment:
I'm pretty sure it won't work, but can we set `spec=` or `autospec=` on this Mock to prevent the typo problem?